models/src/main/protobuf/CasperMessage.proto

```diff
@@ -172,7 +173,8 @@ message BlockMessage {
   int32                  seqNum         = 6; // number of blocks created by the validator
   bytes                  sig            = 7; // signature generated by signing `hash(hash(justification) concat blockHash)`.
   string                 sigAlgorithm   = 8; // name of the algorithm used to sign
-  bytes extraBytes = 9;
+  string                 shardId        = 9; // identifier of the shard where the block was created
```

Comment:
I'm second guessing now whether this belongs here or in the `Header`... It's probably fine here, but maybe logically it belongs in the `Header` or perhaps in `RChainState`...